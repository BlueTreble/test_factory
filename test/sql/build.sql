\set ECHO none
\i test/helpers/psql.sql

BEGIN;
\i test/helpers/deps.sql
\i sql/test_format.sql
ROLLBACK;
