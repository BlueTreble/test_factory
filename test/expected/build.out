\set ECHO none
psql:sql/test_factory.sql:79: NOTICE:  00000: type reference _tf.test_factory.set_name%TYPE converted to text
LOCATION:  LookupTypeName, parse_type.c:143
psql:sql/test_factory.sql:103: NOTICE:  00000: type reference _tf.test_factory.set_name%TYPE converted to text
LOCATION:  LookupTypeName, parse_type.c:143
